spec/unit/bootstrap/bundler_spec.rb

```diff
@@ -125,6 +125,28 @@
           expect(subject).not_to include('--clean')
         end
       end
+
+      context 'with ecs_compatibility' do
+        let(:options) { { :update => 'logstash-integration-jdbc' } }
+
+        it "should allow to update dependencies" do
+          expect(subject).to include('logstash-mixin-ecs_compatibility_support', 'logstash-integration-jdbc')
+
+          subject.each do |gem_name|
```

Comment:
nit: let's name our subjects

> Use named `subject` when possible. Only use anonymous subject declaration when you don’t reference it in any tests, e.g. when `is_expected` is used.
> -- [RSpec Style Guide - Named Subject](https://github.com/rubocop/rspec-style-guide#use-subject)

If we do this up on line 95:

~~~ diff
-subject { LogStash::Bundler.bundler_arguments(options) }
+subject(:bundler_arguments) { LogStash::Bundler.bundler_arguments(options) }
~~~

Then we can refer to it in a way that is more readable:

```suggestion
          bundler_arguments.each do |gem_name|
```